design-toolkit/components/src/components/query-builder/rule.tsx

```diff
@@ -21,35 +21,32 @@ import {
   useRule,
   useStopEventPropagation,
 } from 'react-querybuilder';
-import type { QueryBuilderContextType } from './types';
 import { Lines } from '../lines';
 import { RuleStyles } from './styles';
+import type { QueryBuilderContextType } from './types';
 
-const {
-  lines
-} = RuleStyles();
+const { lines } = RuleStyles();
```

Comment:
Doesn't need to be fixed in this PR, but:
Whoever create this style, don't use slots for single element styles